test/simple.js

```diff
@@ -73,41 +73,43 @@ test.cb('should not delete a fork that has more branches', (t) => {
 	});
 });
 
-// Currently our logic does not delete this fork
-test.skip('should delete a fork that has more branches, but all at upstream branch tips', (t) => {
```

Comment:
Hecks knows, I don’t remember why it works. Perhaps I had added the functionality afterwards and forgot to enable the test? Weird.